pages/common/pigz.md

```diff
@@ -15,6 +15,10 @@
 
 `pigz -0 -p{{4}} {{filename}}`
 
+- Compress a folder using tar:
+
+`tar cf - {{folder}} | pigz > {{filename}}.tar.gz`
```

Comment:
I don't quite understand this command, is this any different from `tar -czf folder.tar.gz folder`? It looks like reinventing the wheel.

Reply:
https://github.com/madler/pigz is a parallelized implementation of gzip, and thus significantly faster at compressing/decompressing large files on multi-core machines.

The existing tldr page for pigz is useful but does not include the de-facto method for compressing folders, which is to tar them first.

Reply:
I see. Thank you for the explanation. So the decompression would still have to be made with `tar`, am I right? I mean, you'd either have to do `pigz -d archive.tar.gz && tar -xf archive.tar` or `tar -xzf archive.tar.gz`, right?

Also, sorry, forgot to mention: it's better to say `directory` instead of `folder`, could you please change that?

Reply:
That's right, and it's worth noting that `pigz` doesn't use multiple cores for decompression. However, it does use them for other calculations which can result in a performance improvement over gzip, [see here](https://stackoverflow.com/a/51688385).

The one-liner to do so would be:
`pigz -dc archive.tar.gz > tar -xf -`